id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/partial_docstr/test_snippet_30.py::test_case_0,functional_correctness_test_folder.post_cut-off.rag.gpt-4.1.partial_docstr.test_snippet_30,test_case_0,functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/partial_docstr/test_snippet_30.py,,xfail,xfailed,,0.00012099999003112316
functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/partial_docstr/test_snippet_30.py::test_case_1,functional_correctness_test_folder.post_cut-off.rag.gpt-4.1.partial_docstr.test_snippet_30,test_case_1,functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/partial_docstr/test_snippet_30.py,,,failed,TypeError: AggregatedStats.__init__() got an unexpected keyword argument 'output_tokens',9.187497198581696e-05
